bylaws/009-per-workflow.md

```diff
@@ -0,0 +1,85 @@
+
+For any proposed change to the PER, the Editor must notify the Core Committee via a post to the mailing list of an Intent to Merge a given change.
+
+If no member of the Core Committee objects within seven days, the proposed change is assumed accepted by default.  Any member of the Core Committee may request during that seven-day period that the change be put to a vote.  If any member does so, the change is immediately put to an Acceptance Vote to determine if it is accepted or not.
+
+The Editor is empowered to declare tagged versions of a PER at any time.  Tagged versions MUST follow Semantic Versioning conventions.  However, any change that would necessitate a new Major release MUST be put to an Acceptance Vote by the Core Committee.
+
+## Staffing Changes
+
+The Editor of an approved PER is empowered to add or remove members from the Working Group at any time, with notification to the Core Committee.
+
+Should the Editor of a PER Working Group resign or become inactive, the Core Committee may appoint a new Editor.  Preference will be given to the existing Working Group members at that time.
+
+## Abandoned
+
+A PER Working Group can be marked as Abandoned by Secretaries when it is without an Editor for 60 days. After a period of 6 months without significant activity in a Working Group, the Secretaries may also declare a PER Working Group to be Abandoned.
+
+Should in the future a new Editor wish to establish a new Working Group to continue development of the same PER, the candidate may petition the Core Committee to do so by asking for a new Entrance Vote.  Only the Editor and two additional individuals are required at this time.
+
+## PER Working Group Fast-track
+
+A PER Working Group may be established with a single Entrance Vote and only three members (Editor and two others) in the following circumstances:
+
+1. A new Working Group is taking over an Abandoned PER.
+2. A new Working Group wishes to use an existing PSR as the initial version of a PER.  In this case, the assumption is that the subject matter is more suited to a PER than a PSR.
+3. A new Working Group wishes to take ownership of existing utility libraries already published that do not yet have clear ownership.
+
+Should a PSR be approved for which its Working Group has already produced one or more utility libraries, the PSR Working Group is automatically converted into a PER Working Group upon approval to cover those libraries, unless the Working Group explicitly states its intent to not do so.  For example, such utility libraries may be better managed by an existing PER for a related topic.
```

Comment:
This section is confusing. I understand that the intention is for a PSR working group to be converted to a PER working group after the approval of the PSR, but the wording is rather circuitous.

Reply:
Yeah, there's a lot of conditionals in this process.  The intent is that, for instance, should PSR-5 ever actually get finished, it would be a PSR that defines parsing rules.  Then after it passes it would mutate into a PER WG to define the tag library.  Or, PSR-18 may have produced a util library, but there's little reason to have a separate PER WG to maintain that when there's already the one left over from PSR-7/15/17 and the PSR-18 people want to just dump the PSR-18 code artifacts on the existing WG to handle (which is probably the best course of action).

If we do a deeper refactor as the list seems to be suggesting I will try to flesh this out further.